Sources/MountebankSwift/Models/Stub/Response/Body.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+private var jsonEncoder = JSONEncoder()
+
+public enum Body: Equatable {
+    public enum Mode: String, Codable, Equatable {
+        case text
+        case binary
+    }
+
+    case text(String)
+    // case textAsData(Data)
```

Comment:
What is the reason this case `textAsData` is commented? Should we implement it?

Reply:
Ah yeah, at some point I thought it would be a nice shorthand for when you read a text file from disk. But you don't really need it. You can just send the data to mountebank as binary, or use `String(data: data, encoding: .utf8)`

Reply:
Removded